managed/services/grafana/client.go

```diff
@@ -43,8 +43,12 @@ import (
 	"github.com/percona/pmm/utils/grafana"
 )
 
-// ErrFailedToGetToken means it failed to get user's token. Most likely due to the fact user is not logged in using Percona Account.
-var ErrFailedToGetToken = errors.New("failed to get token")
+var (
+	// ErrFailedToGetToken means it failed to get user's token. Most likely due to the fact user is not logged in using Percona Account.
```

Comment:
Should it be Percona Account or Grafana account?

Reply:
As I said to Alex https://github.com/percona/pmm/pull/3420/files#r1908599241
This is not related to my changes, but it is between changes because I moved this error inside var().